### ISSUE TITLE
[Aftershock: Exoplanet] Escape pod portal doesn't work, spawned shuttle beacon doesn't either in some circumstances

### ISSUE THREAD

**Issue by carlarctg** [CleverRaven/Cataclysm-DDA#71417]
### Describe the bug

Entering an escape pod portal with a default challenge character and new default map causes an error, 'missing old character in overmap'. This is pretty much a softlock and you can't leave without debug.

Spawning a shuttle beacon on a different, earlier world also was not working for me. I spawned in a new world with the same parameters as above and it worked. I checked overmap and the landing pad building was around there. I'm pretty sure this means that the landing pad building is necessary for shuttle beacons to work, which might cause annoyances when debugging port august moon? It should likely find and create the landpad in that case.

### Attach save file

N/A

### Steps to reproduce

Escape pod bug:

1. Create default world with A:E.
2. Join with default character in escape challenge.
3. Try to portal out, you can't. Die.

Shuttle beacon bug:

1. Create a world with A:E.
2. Spawn in and get an overmap without the landing pad.
3. Spawn a shuttle beacon, watch as the shuttle refuses to send you to port august moon despite giving the message of success.

### Expected behavior

Escape pods should definitely work.

The shuttle beacon should always send you to port august moon. If it's not in the map at the moment it should create it.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3930 (22H2)
- Game Version: 0f39e72 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Aftershock [aftershock],
    Aftershock: Exoplanet [aftershock_exoplanet]
]

**Comment by John-Candlebury** [comment]
What do you mean with "Portal Out"? Did you head to the escape pod (the functional one further from the entrance, not the other one) and didnt teleport to an escape pod on the ground?

The landing pad doesn't matter for the shuttles. I'll need the save for that to check what the problem is. If you got a message claiming that Port Augustmoon couldnt be found on character start, that's probably the problem.

**Comment by Icantthinkofanameforthis** [comment]
would say confirmed for me as well like ya when you pick the scenario: 'Emergency Evacuation' open the escape pod and try get in it gives this error and your stuck on the ship waiting for the 7 min timer to kill you instead of being dropped off on the planets surface

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/70473120/7d6b2847-d22e-4f06-adb9-7802bff93b2f)

also heres a save right before stepping into the escape pod for testing just move south into it
[testing aftershock-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/14142843/testing.aftershock-trimmed.tar.gz)

**Comment by carlarctg** [comment]
@John-Candlebury 

Here's the savefile with a debug character next to the portmaster. I've modded the game slightly to my own ends, which may cause some errors on loading, but I've changed absolutely nothing that isnt item or crafting definitions. All my changes are contained in my 'My Mod' folder so I'm 100% certain of it. You will find a mound of rubble next to the portmaster which happened when I bought a trip to Port Augustmoon.

[Brockport-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/14149052/Brockport-trimmed.tar.gz)
